src/InfoPanel/InfoPanel.less

```diff
@@ -21,25 +22,47 @@
   }
 }
 
-.InfoPanel--header {
-  .border--bottom--s(@neutral_silver);
+.InfoPanel--collapsible {
```

Comment:
Can probably remove this?

Reply:
Ah yeah, I added that in so we could find this class in the unit test, otherwise I felt like I was being too specific on how to drill down into the DOM. I'm open to suggestions on a better way to do this though!

Reply:
Gotcha - so the class's existence makes sense but if you delete these two lines, 25 and 26, does it pose a problem?  I _think_ you shouldn't need them?

Reply:
Huh - when I load your branch locally, these lines are gone.  I may be missing something but in any case, the lines that I originally commented on and seemed extraneous were:

```
.InfoPanel--collapsible {
}
```